src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -1935,13 +1942,21 @@ private void OnLoadedEditorReady()
 
         // Make absolutely sure the current game doesn't have an auto-evo run
         CurrentGame.GameWorld.ResetAutoEvoRun();
+
+        gui.UpdateReportTabStatistics();
     }
 
     private void ApplyAutoEvoResults()
     {
         GD.Print("Applying auto-evo results");
         CurrentGame.GameWorld.GetAutoEvoRun().ApplyExternalEffects();
 
+        CurrentGame.GameWorld.Map.UpdateGlobalTimePeriod(CurrentGame.GameWorld.TotalPassedTime);
+
+        // Needs to be before the remove extinct species call, so that the extinct species
+        // could still be stored for reference (e.g. displaying it as zero on the species population chart)
+        CurrentPatch.RecordConditions();
```

Comment:
Shouldn't this record for all patches (or at least all where the player species is)? Otherwise on moving to a new patch the history there is empty.